wikimedia/wikipedia-ios
# Gif architecture

**Phabricator:** 
https://phabricator.wikimedia.org/T381331
https://phabricator.wikimedia.org/T380802 (accidentally included the work here!)

### Notes
* I had trouble getting the announcement to show up for me logged in because you need to reset it to see the announcement, but the code is in there for anyone who gets it the regular way

### Test Steps
1. In the proper region and language, with a logged in user with personalized slides
2. Load 6+ articles into history then background and foreground
3. Navigate to profile and then year in review
4. Compare images/gifs and copy to Figma, ensure correct.
5. Repeat with logged out user.
6. Check feature announcement while repeating with logged out user, ensure it is updated. 

TLDR:
With both logged in (with personalized slides) and logged out, run through the YIR and compare to Figma
Do the same with feature announcements

### Screenshots/Videos

https://github.com/user-attachments/assets/6c08a63a-3b4b-4425-905c-7109b60e4be8


https://github.com/user-attachments/assets/44a4ce92-c943-4ad9-942b-365b390e4992



![simulator_screenshot_C4C27C5D-A498-4526-8789-748AE1C2DC06](https://github.com/user-attachments/assets/96431c46-7ab1-41c2-b425-d939e672746e)
